tests/testthat/test-use-lintr.R

```diff
@@ -0,0 +1,24 @@
+context("use_lintr")
+
+test_that("use_lintr('local') creates build-ignored, promised file", {
+  scoped_temporary_package()
+  use_lintr("local")
+  expect_true(is_build_ignored("^\\.lintr$"))
+  expect_proj_file(".lintr")
+})
+
+test_that("use_lintr('travis') creates build-ignored, promised file", {
+  scoped_temporary_package()
+  use_lintr("travis")
+  expect_true(is_build_ignored("^\\.lintr$"))
+  expect_proj_file(".lintr")
+})
+
+# test_that("use_lintr('test') creates build-ignored, promised files", {
```

Comment:
This test cannot be run because the `testthat` unit test setup option for `use_lintr` still depends on the old, unacceptable symlink approach. I am hoping someone knows of an alternative approach. Obviously the symbolic link cannot be created without elevated privileges.

Separately, I am unsure how to test the writing out of a test file. If I tentatively uncomment this test and elevate my privileges to enable writing of the symbolic link, the test subsequently fails because of an inability to write out the test file during testing.